docs/trex_release-notes.md

```diff
@@ -8,12 +8,18 @@ layout: docs
 * TOC
 {:toc}
 
+### Tableau 2018.2 (Release Candidate)
```

Comment:
Shouldn't this be Tableau 2018.2
The (Release Candidate) seems like internal information.

Reply:
Yes. Good point. I changed that to just 2018.2.